Update index.js

BlackFenix2 opened the pull request:
Quick update to fix the package crashing as mentioned in https://github.com/leadstoloyals/node-agcod/issues/3

christiaanwesterbeek commented:
I'm not sure why you get the error, because `let x = new BigNumber(123.4567)` (or `BigNumber(123.4567)`)  is supposed to be the way to create the number according to the [docs](https://mikemcl.github.io/bignumber.js/).

It's probably related to a combination of the way the Bignumber repo was setup and your js-bundler. More people reported the problem. 
- https://github.com/MikeMcl/bignumber.js/issues/199
- https://github.com/dignifiedquire/borc/issues/41

I'm not sure if your fix is introducing a problem with other bundlers...

BlackFenix2 commented:
@christiaanwesterbeek I noticed you dont have a LICENSE.MD in your repo, can i fork this project and just work off that? i have to use async/await in my workspace.

christiaanwesterbeek commented:
I'm fine with that. We can also consider to bring you modernisations back to this repo afterwards and create a 2.x version if it's incompatible with the current 1.x.

BlackFenix2 commented:
@christiaanwesterbeek yeah, this project uses the depreciated request NPM package. i can modernize by switching to axios for http requests.

BlackFenix2 commented:
@christiaanwesterbeek i completed my PR and added a createGiftCardAsync() method to allow promise-based API access through axios.

it should not require a major version change since i just added an API endpoint without breaking the existing API. let me know iif you need anything else.

christiaanwesterbeek commented:
It looks backward compatible, and if you had the same experience. I think I can merge this one in and release a 1.1 on npm which is a minor version increment.

BlackFenix2 commented:
@christiaanwesterbeek sounds good, when can we publish the minor version to NPM?
